docs/building.md

```diff
@@ -46,7 +46,7 @@ to run.
 To use Slint from C++, the following extra dependencies are needed:
 
 - **[cmake](https://cmake.org/download/)** (3.19 or newer)
-- A C++ compiler that can do C++20 (e.g., **MSVC 2019 16.6** on Windows)
+- A C++ compiler that can supports C++20 (e.g., **MSVC 2019 16.6** on Windows)
```

Comment:
Maybe we should mention a minimum supported version of GCC and clang? I think it's GCC 10

Reply:
Sounds good. I'm unsure about clang, but I'll add gcc 10.